app/controllers/settings.js

```diff
@@ -2,16 +2,19 @@ import Ember from 'ember';
 
 export default Ember.Controller.extend({
   envVars: Ember.computed.filterBy('model.envVars', 'isNew', false),
-  cronJobs: Ember.computed.sort('model.cronJobs', function(a, b) {
-    if(a.get('branch.defaultBranch')) {
-      return -1;
-    } else if(b.get('branch.defaultBranch')) {
-      return 1;
-    } else {
-      return a.get('branch.name') > b.get('branch.name');
-    }
-  }),
-  branches: Ember.computed.sort('model.branches', function(a, b) {
+  freeBranches: function() {
```

Comment:
It’s my preference to use property names that clearly describe what they refer to. When you look at this here, it’s easy enough to decode that `freeBranches` is about crons, but in the template, you can see `freeSortedBranches` without any explanation of what is “free” branches. So what do you think of a name like `branchesWithoutCron` or something else that explains it’s cron-related?